docs/classic-ui/install.md

```diff
@@ -0,0 +1,442 @@
+    "property=og:description": "Installation notes for Plone Classic UI"
+    "property=og:title": "Installation notes for pip and buildout based installation paths"
+    "keywords": "Plone, Classic UI, classic-ui, installation, pip, buildout"
+---
+
+(classic-ui-installation-label)=
+
+# Installation
+
+This sections describes how to install Plone 6 Classic UI.
+Examples include the following.
+
+- pip based install method
+- buildout based install method
+
+(classic-ui-installation-pip-label)=
+
+## Quickstart pip based installation
+
+install notes for linux
+
+requirements:
+
+- python3.10 or greater
+- python venv module
+
+in our example we use python3.12
+
+On debian based systems you can install python with following command
+
+```bash
+sudo apt install python3.12 python3.12-dev python3.12-venv
+```
+
+Select a directory of your choice
+
+```bash
+mkdir -p /opt/plone && cd /opt/plone
+```
+
+Create a virtual environment
+
+```bash
+python3 -m venv ./venv
+```
+
+Activate the virtual environment
+
+```bash
+source ./venv/bin/activate
+```
+
+Install Plone and a helper package
+
+```bash
+pip install Plone cookiecutter
+```
+
+```bash
+cookiecutter -f --no-input --config-file ./instance.yaml https://github.com/plone/cookiecutter-zope-instance
+```
+
+Deactivate the virtual environment
+
+```bash
+deactivate
+```
+
+minimal content for the `instance.yaml` file
+
+```yaml
+# please change the password to a secrue token!
+default_context:
+  initial_user_name: "admin"
+  initial_user_password: "admin"
+  wsgi_listen: "localhost:8080"
+  debug_mode: false
+  verbose_security: false
+  db_storage: "direct"
+  environment: {
+    "zope_i18n_compile_mo_files": true,
+  }
+```
+
+Start the instance for quick test
+
+```bash
+./venv/bin/runwsgi -v instance/etc/zope.ini
+```
+
+Your instance starts in foreground mode, which is only advisable for troubleshooting or for local demonstration purposes,
+
+Now you can call the url `http://localhost:8080` in your browser and you can add a **Classic UI Plone site**
+
+Let's have fun with Plone!
+
+### Example to start the instance via systemd
+
+ The following systemd service configuration works with the runwsgi script. It assumes your installation is located at /opt/plone and the user account your Plone instance runs under user *plone*:
+
+```ini
+[Unit]
+Description=Plone instance
+After=network.target
+
+[Service]
+Type=simple
+User=plone
+ExecStart=/opt/plone/bin/runwsgi /opt/plone/instance/etc/zope.ini
+KillMode=control-group
+TimeoutStartSec=10
+TimeoutStopSec=10
+
+[Install]
+WantedBy=multi-user.target
+```
+
+Save this configuration under /etc/systemd/system/plone.service and execute `systemctl daemon-reload` for systemd to read it. After that you can use standard `systemctl` commands to `start`, `restart` or `stop` the Plone instance:
+
+```
+systemctl start plone
+systemctl restart plone
+systemctl status plone
+systemctl stop plone
+```
+
```

Comment:
as an alternative you can use also the service command:
```
service plone start
service plone stop
service plone status
service plone restart
```

Reply:
Why would someone use `service *` versus `systemctl *`?